Source/Evolve/WarpXComputeDt.cpp

```diff
@@ -22,17 +22,19 @@
 #include <AMReX_Print.H>
 #include <AMReX_REAL.H>
 #include <AMReX_Vector.H>
-
+#include <AMReX_AmrMesh.H>
 #include <algorithm>
 #include <memory>
 
 /**
  * Determine the timestep of the simulation. */
 void
 WarpX::ComputeDt ()
-{
+{   int Finelev;
+    if (!FinelevInit_flag) Finelev = max_level;
```

Comment:
I would suggest adding a comment to clarify why this is needed.

Reply:
I don't think this function needs to be modified.  It should be OK to keep the size of dt fixed at `max_level+1`.

Reply:
HI, I choose to use Finelev here to print only the value of dt of the coarsen level when the patch is removed. If we keep max_level+1, it will print the value of dt on both levels.

Reply:
We can still keep all the dts without printing levels no longer exist.